src/main/resources/patch_3.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE islands ADD COLUMN extra_value DOUBLE PRECISION;
```

Comment:
Shouldnt this be called _mysql? and where is the sqllite version?

Reply:
It works for both MySQL and SQLite. The two different patches for MySQL and SQLite are optional

Reply:
SQLite doenst have the ALTER TABLE stuff though does it?
isnt that why we added the whole two different systems?

Reply:
and why we had to do this mess for island names?
```sql
CREATE TEMPORARY TABLE islands_backup(id, name, home, visit, create_time, experience, color);
INSERT INTO islands_backup SELECT id, name, home, visit, create_time, experience, color FROM islands;
DROP TABLE islands;
CREATE TABLE islands(`id` INTEGER PRIMARY KEY AUTOINCREMENT , `name` VARCHAR , `home` VARCHAR , `visit` BOOLEAN , `create_time` BIGINT , `experience` INTEGER , `color` VARCHAR NOT NULL ,  UNIQUE (`name`));
INSERT INTO islands SELECT id, name, home, visit, create_time, experience, color FROM islands_backup;
DROP TABLE islands_backup;
```

Reply:
Because creating tables works differently, I guess? The syntax checker said it's not possible

Reply:
Im confused? We are adding a new collum into the table, ALTER TABLE keyword doesnt work for sqllite, and I just tested it, this doesnt work. We need to do what we did for IslandNames

Reply:
SQLite also has certain `ALTER TABLE` statements: https://www.sqlite.org/lang_altertable.html